hawkular-inventory-rest-api/pom.xml

```diff
@@ -59,6 +59,18 @@
     </dependency>
 
     <dependency>
+      <groupId>org.hawkular.inventory</groupId>
+      <artifactId>hawkular-inventory-security-spi</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+    <dependency>
```

Comment:
Maybe this could be moved to the `hawkular-inventory-dist` which assembles the full inventory war?